README.md

```diff
@@ -1,6 +1,8 @@
 # European Union
 
-Implements schemas and codes that are specific to the European law.
+Implements fields and codes that are specific to the European law.
```

Comment:
Remove "the".

Reply:
Done in README and extension.json `.description`.